src/System.Runtime/tests/System/HandleTests.cs

```diff
@@ -30,10 +32,21 @@ public static void  RuntimeMethodHandleTest()
         Assert.True(Co6006LateBoundDelegate.iInvokeCount == 1);
     }
 
+    [Fact]
+    public static void  GenericMethodRuntimeMethodHandleTest()
+    {
+        // Make sure uninstantiated generic method has a valid handle
+        MethodInfo mi1 = typeof(Base).GetMethod("GenericMethod");
+        MethodInfo mi2 = (MethodInfo)MethodBase.GetMethodFromHandle(mi1.MethodHandle);
+        Assert.Equal(mi1, mi2);
```

Comment:
Do we make any guarantees about these being the same exact object, or just equality? If the former, this could be tightened by using Same instead of Equal.